docs/endpointMaps/rest-client-v5.md

```diff
@@ -0,0 +1,176 @@
+
+  # Endpoints table 
+
+  This table includes all endpoints from the official Exchange API docs and corresponding SDK functions for each endpoint that are found in  [rest-client-v5.ts](/src/rest-client-v5.ts). You can find all clients in the [source folder](/src).
+  
+  If anything is missing or is wrong, please open an issue or let us know in our [Node.js Traders](https://t.me/nodetraders) telegram group!
+
+  ## How to use table
+
+  Table consists of 4 parts:
+
+  - Function name
+  - AUTH
+  - HTTP Method
+  - Endpoint
+
+  **Function name** is the name of the function that can be called through the SDK. Check examples folder in the repo for more help on how to use them!
+
+  **AUTH** is a boolean value that indicates if the function requires authentication - which means you need to pass your API key and secret to the SDK.
+
+  **HTTP Method** shows HTTP method that the function uses to call the endpoint. Sometimes endpoints can have same URL, but different HTTP method so you can use this column to differentiate between them.
+
+  **Endpoint** is the URL that the function uses to call the endpoint. Best way to find exact function you need for the endpoint is to search for URL in this table and find corresponding function name. 
+
+  ## All endpoints
+  
+  | Function | AUTH | HTTP Method | Endpoint |
+| -------- | ---- | ----------- | -------- |
+| `fetchServerTime()` | no | GET | `/v5/market/time` |
+| `getServerTime()` | no | GET | `/v5/market/time` |
+| `requestDemoTradingFunds()` | YES | POST | `/v5/account/demo-apply-money` |
+| `getKline()` | no | GET | `/v5/market/kline` |
+| `getMarkPriceKline()` | no | GET | `/v5/market/mark-price-kline` |
+| `getIndexPriceKline()` | no | GET | `/v5/market/index-price-kline` |
+| `getPremiumIndexPriceKline()` | no | GET | `/v5/market/premium-index-price-kline` |
+| `getOrderbook()` | no | GET | `/v5/market/orderbook` |
+| `getTickers()` | no | GET | `/v5/market/tickers` |
```

Comment:
Table is working okay, but it is showing multiple ways that you used to map these functions. I guess it should be okay, cant really avoid that. Unless you have an idea.

Reply:
With multiple ways, do you mean naming inconsistency? We'll do better with future fns